src/entities/MenuItem.java

```diff
@@ -0,0 +1,34 @@
+package entities;
+
+public class MenuItem extends BaseEntity<Long>{
+    private Product product;
```

Comment:
I would suggest a more flexible structure. Just create a Tax class and add a list of taxes to product and final product. The menu item is your final product.

If you want to group them, then create a Menu class that will have a list of final products

Reply:
Ok, that makes sense. I'll do that